src/system.h

```diff
@@ -422,6 +422,21 @@ void Panic_(const char * file, int line, const char * fmt, ...) NORETURN
 #define Panic(...) \
     Panic_(__FILE__, __LINE__, __VA_ARGS__)
 
+void OutOfMemoryPanic(UInt size, const char * reason);
```

Comment:
There should be a comment before this function, too! In fact you have one near the implementation. Personally, I think it's more important to have these documentation comments in the header file. But regardless of what one considers better, it's odd to have the comment for one function (`OutOfMemoryPanic`) only in the .c file, and the comment for the next one (`SetTooLargeAllocationHandler`) only in the `.h` file.

Reply:
Isn't `OutOfMemoryPanic` strictly specific to GASMAN? It should be inside an `#ifdef USE_GASMAN` then -- and perhaps even be declared in `gasman.h` or `sysmem.h` or even `gasman_intern.h`, no?